plugins/css.js

```diff
@@ -1,13 +1,14 @@
 import textLoader from "../loaders/text.js";
 import { postcss, postcssImport, postcssNesting } from "../deps/postcss.js";
 
-export default function () {
+export default function (plugins = []) {
   return (site) => {
     const runner = postcss([
       postcssImport({
         path: site.src("_includes"),
       }),
       postcssNesting(),
+      ...plugins,
```

Comment:
What about instead having the nesting be inside the `plugins = []` because some people may not like that. We could move the import plugin, but since that requires some special configuration it makes sense to keep it.

Reply:
I also think nesting should be left out of the default and left up to the user to add. I kept it in my PR considering there may be existing setups relying on nesting.

Reply:
That makes sense.